support-frontend/assets/pages/paper-subscription-landing/components/content/helpers.jsx

```diff
@@ -26,11 +26,13 @@ import {
   ContactPageLink,
   useDotcomContactPage,
 } from 'helpers/dotcomContactPage';
+import InfoSvg from './info.svg';
 
 // Types
 export type ContentPropTypes = {|
   selectedTab: ActiveTabState,
-  setTabAction: typeof setTab
+  setTabAction: typeof setTab,
+  useDigitalVoucher?: Option<boolean>,
```

Comment:
Does this need to be an optional prop **and** `Option<boolean>`?

Reply:
Also is it actually used? I can't see anywhere where it is.

Reply:
No, I don't think we need it to be an optional prop, but we will need it as the text of the switch link needs to be conditional on which collection method it is. It does need to be optional as it can be null or true.